index.html

```diff
@@ -420,6 +426,11 @@ <h3>
           {{RenderBlockingStatusType}} <a data-dfn-for=
           "PerformanceResourceTiming"><dfn>render-blocking status</dfn></a>.
         </p>
+        <p>
+          A <a>PerformanceResourceTiming</a> has an associated
```

Comment:
I thought `service worker timing` will be a member of fetch timing info, and values are filled there.
If not, can I ask how the values are filled?